scripts/sync_tables/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ghcr.io/sage-bionetworks/genie:gen-2230-refactor_update_table
```

Comment:
Can this not run independently of this feature branch's docker image?

Reply:
I forgot that `sync tables` uses genie functions and not really a way around it. Nvm.